### ISSUE TITLE
I got stuck in val，can you give me some suggest?

### ISSUE THREAD

**Issue by DongGuaDONG** [dragonbook/V2V-PoseNet-pytorch#2]
https://github.com/bearpaw/pytorch-pose/issues/37   just like this.

**Comment by dragonbook** [comment]
I have not encountered problems like this. I used PyTorch-0.4.1 + python-3.6 + cuda(92/90/80) with anaconda under ubuntu 16.04. I guess if it is a problem in your PyTorch version or python environment?

**Comment by dragonbook** [comment]
Tested under GTX1070 and TITAN XP.

**Comment by dragonbook** [comment]
As PyTorch [release page](https://github.com/pytorch/pytorch/releases) shows, there are some Data Loader / Datasets / Multiprocessing bug fixes in PyTorch v0.3.1 and later releases. Which PyTorch version do you use?

**Comment by DongGuaDONG** [comment]
I used PyTorch-0.4.1 + python-3.5 + cuda80  under ubuntu 16.04.Tested under GTX1080  and GTX1080Ti，and would you support NYU and ICVL test future?

**Comment by dragonbook** [comment]
I'm not sure what exactly the problem is. But it's may not the bug in repo's code, and here are some debug directions:

1. Environment related:
- disable cudnn for batchnorm(or just use cuda or use cpu) to check, as suggested in repo's README. (related discussions [pytorch-pose/issues/33](https://github.com/bearpaw/pytorch-pose/issues/33), [pytorch-pose-hg-3d/issues/16](https://github.com/xingyizhou/pytorch-pose-hg-3d/issues/16))

- Or, create a new environment with anaconda to see if we can bypass this problem.

2. Configuration related:
- set batch_size = 1, set num_workers=1 to see what happens.

3. Data related:
- use only one or fewer samples to overfit to see what happens.
- use your own small real(or simulated) dataset to see what happens.

**Comment by DongGuaDONG** [comment]
Thanks a lot,I will try it ! And would you support NYU and ICVL test future?

**Comment by dragonbook** [comment]
I have no plan right now, but it should not be hard to port it from the [author's repo](https://github.com/mks0601/V2V-PoseNet_RELEASE).

**Comment by DongGuaDONG** [comment]
https://github.com/pytorch/pytorch/issues/1355 It's a bug.

**Comment by dragonbook** [comment]
That's good!  Can you train your model now?

**Comment by DongGuaDONG** [comment]
Yes

**Comment by DongGuaDONG** [comment]
Thanks a lot！
